server/pbenchinacan/container-build.sh

```diff
@@ -16,6 +15,12 @@ set -o errexit
 BASE_IMAGE=${BASE_IMAGE:-registry.access.redhat.com/ubi9:latest}
 RPM_PATH=${RPM_PATH:-/root/sandbox/rpmbuild/RPMS/noarch/pbench-server-*.rpm}
 
+if (( $(ls ${RPM_PATH} 2>/dev/null | wc -l) != 1 ))
+then
+    echo "RPM_PATH (${RPM_PATH}) does not uniquely identify the RPM file" >&2
+    exit 2
+fi
+
```

Comment:
The problem is that I don't want to create container objects if we are going to fail later.

Reply:
Given that they are not all configuration definitions, at least I did not recognize the groupings you apparently intended, and some of them are derived from external commands, I don't see why we need to move the check away from where `RPM_PATH` is defined.

Reply:
The first group describes the main inputs to the script; the next block describes the locations for the outputs; the next block describes the locations of additional inputs in the Git checkout; and the last one defines the image tag.  They are literally a straight sequence of assignments (and exactly one of them uses an external command, but only when the value isn't provided externally).

There's no need to break that up.  Checking the configuration can wait until we've finished defining it all.  The important thing is to do the check before we've paid any significant cost, and that doesn't start until the first `buildah` invocation.

Reply:
I don't see the benefit of separating the definition from its verification.  The grouping adds less value than the check (which I was burned multiple times and had to cleanup dangling environment stuff).

To avoid a lot of debate, I have created sections to make the desire for this file to be explicit rather than ... discovered.